akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -96,208 +105,222 @@ private[http] object HttpServerBluePrint {
         // `buffer` will ensure demand and therefore make sure that completion is reported eagerly.
         .buffer(1, OverflowStrategy.backpressure)
 
-    // we need to make sure that only one element per incoming request is queueing up in front of
-    // the bypassMerge.bypassInput. Otherwise the rising backpressure against the bypassFanout
-    // would eventually prevent us from reading the remaining request chunks from the transportIn
-    val bypass = Flow[RequestOutput].collect {
-      case r: RequestStart      ⇒ r
-      case m: MessageStartError ⇒ m
-    }
-
     val rendererPipeline =
```

Comment:
minor nitpick: rename `requestParsingFlow`/`rendererPipeline` to `requestParsing`/`responseRendering`?